docs/3-MODELS.md

```diff
@@ -0,0 +1,124 @@
+## 2. MODELS (Storing the collected data)
+
+The data collected from various sources, as discussed in [2-SOURCES](/docs/2-SOURCES.md), is stored in a PostgreSQL
+database. The tables are designed using Django models.
+
+All of the project's models are defined inside the app: `ports`. The models are split into three different files: `port`,
+`buildhistory` and `stats`. Each file contains the models related to the category defined by file name.
+
+#### 1. `Port` Model
+
+It stored the basic information related to a port, with the following fields:
+- `portdir`
+- `description`
+- `homepage`
+- `epoch`
+- `platforms`
+- `categories` - A many-to-many relation with `Category` model
+- `long_description`
+- `version`
+- `revision`
+- `closedmaintainer`
+- `name`
+- `license`
+- `replaced_by` - Default values is `Null` for active ports. If the value is not null, the port is considered as obsolete.
+- `active` - Value is `False` for deleted ports, otherwise `True`*.
```

Comment:
```suggestion
- `active` - `True` by default. `False` when the port no longer exists (in `PortIndex`).
```

Reply:
Could you provide an example (URL pointing to ports.macports.org) of both (one `replaced_by` and one `active=False`) ports? For the `replaced_by` pick some python or perl subport which is simply replaced by a newer subport.